docs/getting-started.rst

```diff
@@ -62,9 +62,11 @@ A few things to note here:
   the test begins and that the container is stopped and removed after the test
   ends.
 - The scope of the fixture is important. By default, pytest fixtures have
-  function scope, which means for each test function the fixture is completely
-  reinitialized. Creating and starting up a container can be a little slow, so
-  you need to think carefully about what scope to use for your fixtures.
+  function scope, which means that for each test function the fixture is
+  completely reinitialized. Creating and starting up a container can be a
+  little slow, so you need to think carefully about what scope to use for your
+  fixtures. See :meth:`~seaworthy.definitions.ContainerDefinition.clean` for a
```

Comment:
Does this render as `ContainerDefinition.clean` or just `clean`? If the latter, we may need to do something fancier with the reference text.

Reply:
The latter :(